homework/index.html

```diff
@@ -0,0 +1,60 @@
+<html>
+  <head>
+    <title>The-Sopranos</title>
+    <link rel="stylesheet" href="styles.css" />
```

Comment:
dodati meta viewport za prilgodbu mobilnim uređajima:
`<meta name="viewport" content="width=device-width, initial-scale=1.0" />`

Ovako bi to sad izgledalo na iPhoneu 14
![Screenshot 2024-07-07 at 15 53 36](https://github.com/infinum-academy/js-react-academy-2024-franracic/assets/108664913/45545bf6-70f0-4092-8396-abc4ac289de3)